<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профсоюз.Льготы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(240, 240, 240)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(254, 127, 94);
color: 'white'</string>
   </property>
   <property name="text">
    <string>     Профсоюз.Льготы</string>
   </property>
  </widget>
  <widget class="QLabel" name="fio_lab">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string>Неизвестно</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="lgots_area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>361</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
border: 1px solid rgb(254, 127, 94);
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>399</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="new_part_lgot">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgb(254, 127, 94);
color: 'white';
border: 2px solid rgb(254, 127, 94);
border-radius: 15px
}

QPushButton:pressed {
background: rgb(234, 107, 74)
}</string>
   </property>
   <property name="text">
    <string>Добавить партнера и льготу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>75</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
border: 2px solid rgb(240, 240, 240);
color: rgb(254, 127, 94)</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
